packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -57,6 +57,19 @@ Written for Sketch 59.1
 4. Turn the new artboards into symbols.
 5. In the artboard names, replace "icon name" with the name of your icon.
 
+
+<Callout>
+   <CalloutTitle>How to name icon artboards in Sketch</CalloutTitle>
+   <CalloutText>
+     Use camel case. For an interface icon, give the icon a name that describes its 
+     purpose rather than its content. For example, "24/linkExternal" instead of 
+     "24/arrowUpRight". If you're adding an icon used to brand a product, name it 
```

Comment:
Can you add a line break before "If you're adding an icon..."

Reply:
Like so? 
![image](https://user-images.githubusercontent.com/1592327/74554812-03d41580-4f20-11ea-9bbc-116afbc6114d.png)